.travis.yml

```diff
@@ -9,13 +9,13 @@ go:
 
 before_install:
   - curl -Lo kind https://github.com/kubernetes-sigs/kind/releases/download/${KIND_VERSION}/kind-linux-amd64 && chmod +x kind && sudo mv kind /usr/local/bin/
-  - curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/${KUBERNETES_VERSION}/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
+  - curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/v${KUBERNETES_VERSION}/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
```

Comment:
Out of curiosity only: Why don't maintain the "v" in the version constant?

Reply:
The var is used also to set the kubernetes version in kind for integration tests and in other case it will break the cluster bootstrap.
https://github.com/spotahome/kooper/blob/master/hack/scripts/integration-test-kind.sh#L6